src/PowerShellEditorServices.Protocol/LanguageServer/Completion.cs

```diff
@@ -69,6 +75,11 @@ public class TextEdit
     [DebuggerDisplay("Kind = {Kind.ToString()}, Label = {Label}, Detail = {Detail}")]
     public class CompletionItem
     {
+        public CompletionItem()
+        {
+            this.InsertTextFormat = InsertTextFormat.PlainText;
```

Comment:
Maybe just use a default parameter value instead of putting it in the constructor?

Reply:
Default parameter value where?  I could just nuke this constructor and every where we use a Completion (just two places), you have to set the InsertTextFormat property.

Reply:
See below!

Reply:
Ah, default *property* value. Heh, I started with that.  :-)  But then it seemed odd to me that the init appeared so far down the class def.  That is one thing I don't like about auto-property initialization.  I guess I'm just used to being able to see class member initialization towards the beginning of a class def.  But I'm happy to make that change if you'd prefer that.